samples/gibbs_ensemble/run_for_several_kT.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+python=$(which python3)
+pypresso=../../build/pypresso
+
+if [[ ! -e $pypresso ]]
```

Comment:
```suggestion
if [ ! -e "${pypresso}" ]
```
This change makes the script Dash-compatible. The quotation marks are required in Dash because the pypresso path may contain whitespaces (especially on Windows systems).